app/livechat/client/views/app/tabbar/externalSearch.js

```diff
@@ -19,9 +20,13 @@ Template.externalSearch.events({
 
 Template.externalSearch.onCreated(function() {
 	this.roomId = null;
-	// console.log('externalSearch.this ->',this);
-	this.autorun(() => {
+	this.externalMessages = new ReactiveVar([]);
+
+	this.autorun(async () => {
 		this.roomId = Template.currentData().rid;
-		this.subscribe('livechat:externalMessages', Template.currentData().rid);
+		if (this.roomId) {
+			const { messages } = await APIClient.v1.get(`livechat/messages.external?roomId=${ this.roomId }`);
```

Comment:
We need to use pagination here, otherwise, only the first fetch will be displayed.